tap_slack/schemas/messages.json

```diff
@@ -150,8 +150,7 @@
       "type": [
         "null",
         "string"
-      ],
-      "format": "date-time"
+      ]
```

Comment:
Necessary to delete the formatting?

Reply:
For whatever reason, the formatting was giving me errors. I removed it and never really looked back. I will revisit and try to determine what the error was and whether we can maintain the formatting.